region/tourian/main/Tourian Escape Room 2.json

```diff
@@ -100,11 +100,92 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharged, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 95
+        }
+      },
+      "requires": [
+        {"or": [
+          "ScrewAttack",
```

Comment:
I don't think screw attack works without another jump... maybe sometimes.
A super to kill the pirate would work.

Reply:
Good catch.